modules/nw-udn-best-practices.adoc

```diff
@@ -25,7 +30,3 @@ Before setting up a `UserDefinedNetwork` (UDN) resource, users should consider t
 
 * The cluster subnet and services CIDR for a UDN cannot overlap with the default cluster subnet CIDR. OVN-Kubernetes network plugin uses `100.64.0.0/16` as the default network's join subnet, you must not use that value to configure a UDN `JoinSubnets` field.
```

Comment:
aren't we using 100.65.0.0/16 for the primary UDN ? 

We should call that out as the default for UDN, and that the customer must override the join subnets if their subnets somehow overlap w/ that one. 

@tssurya to keep me honest here.

Reply:
I updated with a note that this must be overridden if your cluster's network uses that address anywhere.

Reply:
From what I know, since default join-subnet may change we avoided specifying it

Reply:
But, as a user, how do I know if I need to override the value ??? 

The admin(s) will know the ranges used in their networks, but if don't know what is the value being used here, they have no chance of knowing when they need to define a different join address.